recipe/patch_yaml/cudnn.yaml

```diff
@@ -8,3 +8,116 @@ if:
   subdir_in: win-64
 then:
   - remove_depends: "libzlib-wapi *"
+---
+# As a consequence of https://github.com/conda-forge/cudnn-feedstock/issues/124 and
+# https://github.com/conda-forge/admin-requests/pull/1687, we need to loosen the
+# run-exports for cudnn-dependent builts (on CUDA 12.x) to avoid versions marked broken.
+if:
+  has_depends: "*cudnn*"
+  not_has_depends: cuda-version >=13.0,<14
+  not_has_constrains: cuda-version >=13.0,<14
+  timestamp_lt: 1759943917000
```

Comment:
The timestamps should be updated to now before this is merged. Just in case any packages were rebuilt.

Reply:
```suggestion
  timestamp_lt: 1761236193526
```